internal/juju/secrets.go

```diff
@@ -52,7 +52,7 @@ type ReadSecretInput struct {
 }
 
 type ReadSecretOutput struct {
-	Value map[string]string
+	SecretDetails apisecrets.SecretDetails
```

Comment:
You need the name, info and the values here. Otherwise import won't work correctly.

Reply:
Got you. Done